main/reference/zoe-api/zoe-helpers.md

```diff
@@ -19,9 +19,55 @@ ZoeHelper functions are contract helpers, in that they are useful to contract co
 Contracts are started up by Zoe, 
 and **zcf** is passed in as a parameter to **start()**. 
 
+## atomicRearrange(zcf, transfers)
+- **zcf**: **[ZoeContractFacet](./zoe-contract-facet.md)** 
+- **transfers**: **Array&lt;[TransferPart](./zoe-data-types.md#transferpart)>**
+- Returns: None.
+
+Asks Zoe to rearrange the **[Allocations](./zoe-data-types.md#allocation)** among the seats
+mentioned in *transfers*. *transfers* are a set of changes to **Allocations** that must satisfy
+several constraints. If these constraints are all met, then the
+reallocation happens atomically. Otherwise an error is thrown and it does not happen
+at all. The constraints are as follows.
+
+ * All the mentioned seats are still live.
+ * There aren't any outstanding stagings for any of the mentioned seats.
+
+	Stagings are a reallocation mechanism that has been
+    deprecated in favor of this **atomicRearrange()** function.
+    To prevent confusion, each reallocation can only be
+    expressed in the old way or the new way, but not a mixture.
+ * Overall conservation must be maintained. In other words, the reallocated
+    **[Amounts](/reference/ertp-api/ertp-data-types.md#amount)** must balance out to zero.
+
+Note that you can manually construct the **TransferParts** that make up the *transfers* array manually,
+or you can use the helper functions **[fromOnly()](#fromonly-fromseat-fromamounts)** or 
+**[toOnly()](#toonly-toseat-toamounts)** to create simple **TransferParts** that only contain 
+two of the possible 4 potential fields.
+
+## fromOnly(fromSeat, fromAmounts)
+- **fromSeat**: **[ZCFSeat](./zcfseat.md)**
+- **fromAmounts**: **[AmountKeywordRecord](./zoe-data-types.md#amountkeywordrecord)**
+- Returns: **[TransferPart](./zoe-data-types.md#transferpart)**
+
+Returns a **TransferPart** when a reallocation only involves taking an 
+**[Allocation](./zoe-data-types.md#allocation)** from the **ZCFSeat** specified by *fromSeat*.
+**TransferParts** are used as part of the *transfer* argument of the 
+**[atomicRearrange()](#atomicrearrange-zcf-transfers)** function.
+
+## toOnly(toSeat, toAmounts)
+- **toSeat**: **[ZCFSeat](./zcfseat.md)**
+- **toAmounts**: **[AmountKeywordRecord](./zoe-data-types.md#amountkeywordrecord)**
+- Returns: **[TransferPart](./zoe-data-types.md#transferpart)**
+
+Returns a **TransferPart** when a reallocation only involves giving an 
+**[Allocation](./zoe-data-types.md#allocation)** to the **ZCFSeat** specified by *toSeat*.
+**TransferParts** are used as part of the *transfer* argument of the 
+**[atomicRearrange()](#atomicrearrange-zcf-transfers)** function.
+
```

Comment:
Please add the `atomicTransfer` helper here. It's useful when the entire transfer can be represented by a single tranferPart, and we'll probably keep it even if we ever get around to dropping the atomicRearrange  helper.

Reply:
Documentation for `atomicTransfer` has been added.